kong/plugins/log-serializers/basic.lua

```diff
@@ -12,11 +12,12 @@ function _M.serialize(ngx)
       consumer_id = ngx.ctx.authenticated_credential.consumer_id
     }
   end
-  
+
   return {
     request = {
-      uri = ngx.var.request_uri,
-      request_uri = ngx.var.scheme .. "://" .. ngx.var.host .. ":" .. ngx.var.server_port .. ngx.var.request_uri,
+      request_uri = ngx.var.request_uri,
+      uri = ngx.var.uri,
+      request_url = ngx.var.scheme.."://"..ngx.var.host..":"..ngx.var.server_port..ngx.var.request_uri,
```

Comment:
Yes, this is a breaking change indeed. The whole point is to name the variables same as the Nginx variables to avoid confusion. Also, if we are mentioning parameters like scheme and host, it makes more sense to rename it to `request_url`, i.e `url` which is more specific as opposed to `uri`, no?